src/dapla_metadata/datasets/core.py

```diff
@@ -491,6 +491,10 @@ def write_metadata_document(self) -> None:
         self.dataset.metadata_last_updated_by = (
             user_info.get_user_info_for_current_platform().short_email
         )
+        try:
+            self.dataset.owner = user_info.get_owner()
```

Comment:
We should also set this when we open a dataset so that the user can see that it's filled out